packages/browser/package-lock.json

```diff
@@ -8,6 +8,7 @@
       "version": "7.10.4",
       "resolved": "https://registry.npmjs.org/@babel/code-frame/-/code-frame-7.10.4.tgz",
       "integrity": "sha512-vG6SvB6oYEhvgisZNFRmRCUkLz11c7rp+tbNTynGqc6mS1d5ATd/sGyV6W0KZZnXRKMTzZDRgQT3Ou9jhpAfUg==",
+      "dev": true,
```

Comment:
This is suspicious again - no changes to `package.json`, but lots of `"dev": true` added.

Reply:
I'm really not sure why this keeps happening... I assume it is somehow related to Lerna, but I don't know what causes this.